eo-runtime/src/main/java/EOorg/EOeolang/EObytes$EOleft.java

```diff
@@ -25,23 +25,29 @@
 package EOorg.EOeolang;
 
 import org.eolang.*;
+import java.math.BigInteger;
 
 /**
- * LEFT.
+ * BYTES.LEFT.
  *
  * @since 1.0
  */
-@XmirObject(oname = "int.left")
-public class EOint$EOleft extends PhDefault {
+@XmirObject(oname = "bytes.left")
+public class EObytes$EOleft extends PhDefault {
 
-    public EOint$EOleft(final Phi sigma) {
+    public EObytes$EOleft(final Phi sigma) {
         super(sigma);
         this.add("x", new AtFree());
-        this.add("φ", new AtComposite(this, self -> new Data.ToPhi(
-            new Dataized(self.attr("ρ").get()).take(Long.class)
-            <<
-            new Dataized(self.attr("x").get()).take(Long.class)
-        )));
+        this.add("φ", new AtComposite(this, self -> {
+            final byte[] array = new Dataized(
+                    self.attr("ρ").get()
```

Comment:
@IngeniariusSoftware just curious, why do you use 8 spaces in your code if everywhere in the code we use 4?